### PR TITLE
Better matched the CC button for the YT theme

### PR DESCRIPTION
**Pull request by heff**
image
image
image

### CODE REVIEW

**Review by mmcc — APPROVED** [review]
💅

File: themes/yt/template.html [hunk 1]
```diff
@@ -90,6 +90,41 @@
...
+    display: block;
+    position: absolute;
+    width: 0;
+    height: 3px;
+    border-radius: 3px;
+    background-color: #f00;
+    bottom: 6px;
+
+    transition: left 0.25s cubic-bezier(0, 0, 0.2, 1),
+      width 0.25s cubic-bezier(0, 0, 0.2, 1);
```
Comment (luwes) [inline themes/yt/template.html:122]:
```suggestion
    transition: left .1s cubic-bezier(0.4, 0, 1, 1), 
      width .1s cubic-bezier(0.4, 0, 1, 1);
```

the hiding is faster on YT

File: themes/yt/template.html [hunk 2]
```diff
@@ -90,6 +90,41 @@
...
+  }
+
+  media-captions-button[mediasubtitleslist][aria-checked="true"]:after {
+    width: 24px;
```
Comment (luwes) [inline themes/yt/template.html:126]:
```suggestion
    width: 24px;
    transition: left 0.25s cubic-bezier(0, 0, 0.2, 1),
      width 0.25s cubic-bezier(0, 0, 0.2, 1);
```

**Comment by luwes** [comment]
just one nit. otherwise looks good!

**Comment by heff** [comment]
Added Wes's changes and a shadow to the svg.

**Pull request merged**